certified-connectors/DocuMotor/apiDefinition.swagger.json

```diff
@@ -0,0 +1,113 @@
+    "version": "1.0",
+    "contact": {
+      "name": "DocuMotor Support",
+      "url": "https://support.omnidocs.com/hc/en-us/requests/new",
+      "email": "support@documotor.com"
+    }
+  },
+  "host": "documotor.com",
+  "basePath": "/api",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://omnidocs.com/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://omnidocs.com/privacy-policy-2"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Productivity;Content and Files"
+    }
+  ],
+  "paths": {
+    "/template/{templateId}/generate": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "ok",
+            "x-ms-visibility": "important"
+          }
+        },
+        "produces": [
+          "application/pdf",
+          "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+          "application/vnd.openxmlformats-officedocument.presentationml.presentation",
+          "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
+        ],
+        "parameters": [
+          {
+            "x-ms-summary": "Template ID",
+            "in": "path",
+            "type": "string",
+            "name": "templateId",
+            "x-ms-url-encoding": "single",
+            "required": true,
+            "description": "The ID of the template."
+          },
+          {
+            "x-ms-summary": "Accept Type",
+            "in": "header",
+            "name": "Accept",
+            "type": "string",
+            "required": true,
+            "description": "Output file type. */* denotes the native template format.",
+            "enum": [
+              "*/*",
+              "application/pdf"
+            ]
+          },
+          {
+            "x-ms-summary": "Stage ID",
+            "in": "header",
+            "name": "stageId",
+            "type": "string",
+            "required": false,
+            "default": null,
+            "description": "Required for versioned templates, must be blank for non-versioned templates."
+          },
+          {
+            "x-ms-summary": "Document Data",
+            "in": "body",
+            "name": "Document data",
+            "description": "The dataset used to generate the document.",
+            "schema": {
+              "type": "object"
+            }
+          }
+        ],
+        "summary": "Generate a document",
+        "description": "A document is generated based on the input data and template id.",
+        "x-ms-visibility": "important",
+        "operationId": "Generate-doc"
```

Comment:
@sebastianehlers all operationIds should be in Camel Case. Please change to "GenerateDoc".

Reply:
Done.